frontend/src/metabase/status/components/FileUploadStatusLarge/FileUploadStatusLarge.tsx

```diff
@@ -91,16 +91,19 @@ const getName = (upload: FileUpload) => {
   return upload.name;
 };
 
-const getTitle = (uploads: FileUpload[], collection: Collection) => {
+const getTitle = (
+  uploads: FileUpload[],
+  uploadDestination: Collection | Table,
+) => {
   const isDone = uploads.every(isUploadCompleted);
   const isError = uploads.some(isUploadAborted);
 
   if (isDone) {
-    return t`Data added to ${collection.name}`;
+    return t`Data added to ${uploadDestination.name}`;
   } else if (isError) {
-    return t`Error uploading your File`;
+    return t`Error uploading your file`;
   } else {
-    return t`Uploading data to ${collection.name} …`;
+    return t`Uploading data to ${uploadDestination.name} …`;
```

Comment:
The space stood out to me. Why not "Uploading data to TableName..."?

Reply:
sometimes tableId can be like `card_1234` if we're using another question as a data source 🥴